src/GslCore/TaggingProvider.fs

```diff
@@ -56,11 +56,14 @@ let gTagPragmaDef =
 let foldInTags (cmdlineTags : AssemblyTag list) (_at : ATContext) (a : DnaAssembly) =
     // gtag is global tag, tag is dna assembly tag
     match List.collect (fun pragma -> pragma.args) ([ a.pragmas.TryFind("tag") ; a.pragmas.TryFind("gtag") ] |> List.choose id) with
-    | [] -> ok a
+    | [] ->
+        let newTags = cmdlineTags |> Set.ofList |> Set.union a.tags
```

Comment:
Fixing an issue here where `TaggingProvider` was not responding to `--tag` CLI parameter